dist.ini

```diff
@@ -10,13 +10,17 @@ version = 0.01
 -bundle = @Basic
 -remove = GatherDir
 [Git::GatherDir]
+exclude_match = ^script\/
+[Run::AfterBuild]
+run = mkdir %d/script/ && cp %o/script/gen_module_docs.pl %d/script/ && cd '%d' && script/gen_module_docs.pl && rm script/gen_module_docs.pl
```

Comment:
This should definitely be rolled into a separate appendix .pod - it adds ~10k lines to the module.